jina/hubble/helper.py

```diff
@@ -83,16 +83,13 @@ def get_request_header() -> Dict:
 
     return headers
 
+def is_logged_in() -> bool:
+    """Return the status of user logged
 
-def is_logged_in() -> bool: 
-    """Return true if user logged in else false.
-
-    :return: bool  
-    """  
-      
-    auth_token = get_token()
-    return True if auth_token else False
+    :return: bool
+    """
 
+    return hub_is_logged_in()
```

Comment:
Why not removing this at all?

Reply:
Shouldn't we wrap it for third-party dependencies?

Reply:
If so, that means you will have thousands of functions that just wrap third-party function. Does that really make sense?